lib/main_common.pm

```diff
@@ -1185,7 +1185,7 @@ sub load_consoletests {
     loadtest "console/zypper_log";
     if (!get_var("LIVETEST")) {
         loadtest "console/yast2_i";
-        loadtest "console/yast2_bootloader";
+        loadtest "yast2_cmd/yast2_bootloader" if (is_opensuse);
```

Comment:
What's about QAM? Is it schedule for other versions of SLES?

Reply:
Sure. let me add this. my neglect

Reply:
is_sle('<15+') will do, right?

Reply:
Nope, because we will have SP3 and need to apply for SP3 only. So, use solution @jknphy has used. We only want to exclude module from functional job group.

Reply:
i dont understand what `to apply for SP3 only` means here. if i can understand maybe i can think something else. the reason is because i think that the other solution is not very easy maintainable. ofc if not other better solution found, then the variable is easy to apply

Reply:
i think i got it. let me take a look

Reply:
EXCLUDE_MODULES is used

Reply:
But still, "if (is_opensuse)" will affect QAM tests that use the module. Why do you need a  condition here, since you will use "EXCLUDE_MODULES" in functional job groups?

Reply:
Ah, sorry, outdated